README.md

```diff
@@ -370,9 +370,35 @@ print(tile.shape)
 > (1, 256, 256)
 ```
 
-#### Working with multiple assets
+## Working with multiple assets
 
-`rio_tiler.io.cogeo` submodule has `multi_*` functions (tile, part, preview, point, metadata, info, stats) allowing to fetch and merge info 
+#### Mosaic
+
+Starting in rio-tiler 2.0, we've transfered the [rio-tiler-mosaic](https://github.com/cogeotiff/rio-tiler-mosaic) plugin to be a rio-tiler submodule.
```

Comment:
`transferred`. I'd also suggest saying "a rio-tiler module", rather than "submodule" because the latter makes me think of a Git submodule. I was actually confused before that the code was all copied in, because from the title I was expecting a git submodule